web/src/drafts_overlay_ui.ts

```diff
@@ -342,6 +406,8 @@ export function update_bulk_delete_ui(): void {
         }
     } else {
         if ($unchecked_checkboxes.length > 0) {
+            $select_drafts_button.show();
+            $delete_selected_drafts_button.show();
```

Comment:
You said this was added specifically for when the user clicks "undo", should it live in `undo_draft_deletion` then?